### ISSUE TITLE
Timezone not taken into account for schedules

### ISSUE THREAD

**Issue by RKLBusinessDevelopment** [Volmarg/personal-management-system#136]
When I add a new schedule in the calendar, everything looks fine and timezone is taken into account (eg. current time 01:30 -> calendar shows 01:30). However, once the entry is saved, the timezone is wrong (UTC) so as a workaround I have to add +2 hours for every schedule I need to create.

Is this a bug or did I miss some configuration setting? I tried setting TZ=Europe/Berlin in the environment section of the php-fpm container and also mapped the /ect/localtime:/ect/localtime volume but that didn't fix the problem.

Thanks for a great tool which would be even better with the timezone problem solved. All pointers are welcome. :)

**Comment by Volmarg** [comment]
That's a bug, it's something on front. 
The dates in calendar are proper, but the data sent to backend has wrong data.

This will sadly need to remain a bug for now, because i can't even quickly run a dev .env on the project anymore.

I'm working on rework and not sure if i will be using the same calendar anymore or not (tho it will take months to finish - https://github.com/Volmarg/personal-management-system-front)

Basically the bug is somewhere in `TuiCalendarService.ts`, but upon changes the `npm run build` has to be called.

**Comment by RKLBusinessDevelopment** [comment]
Thank you for the information and good luck with the rewrite. Meanwhile, I can easily add the +2 hours difference. Good to hear that you are still working on this project, I really appreciate that.

**Comment by Volmarg** [comment]
Is fixed in rewrite - working on calendar now and had the same problem.

It will still take months before all is done, but the issue will gone then.
